src/fides/api/models/privacy_request.py

```diff
@@ -680,28 +673,24 @@ def get_paused_collection_details(
     def cache_failed_checkpoint_details(
         self,
         step: Optional[CurrentStep] = None,
-        collection: Optional[CollectionAddress] = None,
```

Comment:
We're not actually using the collection to cache failed checkpoint details anywhere so this is being removed altogether